selfhost/raw.wyv

```diff
@@ -1,5 +1,6 @@
 module raw
 
+import types
```

Comment:
You probably want one representation of Type for raw, which just represents the type as a String, and a different representation of Type for bound, which uses a Binding to internally represent the Type.